spec/01-lexical-syntax.md

```diff
@@ -553,6 +553,20 @@ but are required to be properly nested.  Therefore, a comment like
 `/* /* */` will be rejected as having an unterminated
 comment.
 
+## Multi-line and trailing commas
+
+If a comma is followed by a line end and then a right parenthesis, square
+bracket or curly brace (`)`, `]`, and `}`, respectively) then such a trailing
+comma is ignored. For example:
```

Comment:
"immediately" seems useful to say, but this is modulo spaces. I wanted to say "token" somewhere to emphasize this.